src/Frontend/Core/Js/Vue-components/VShareButton.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
```

Comment:
Would it be an idea to add a `data-role="share-button"` so we can access it later on through JS? Or is this overengineering?
Or add a class on the wrapper so it can be styled more easily?

Reply:
I think that is overkill. Besides that I think there are other ways to access the component. The styling is on de buttons its self. I should not add an unnecessary class if not needed. Just to keep it simple.

Reply:
Agreed